BraintreeModels/src/main/java/com/braintreepayments/api/TokenizeCallback.java

```diff
@@ -0,0 +1,14 @@
+package com.braintreepayments.api;
+
+
+/**
+ * Communicates JSON response from a tokenization request on the main thread.
+ */
+interface TokenizeCallback {
+
+    /**
+     * @param tokenizationResponse parsed {@link BraintreeNonce} from the HTTP request.
```

Comment:
I think this could clarified - this string is the raw JSON of the tokenization response as a string?

Reply:
We re-worded this